src/clarity-angular/button/_toggles.clarity.scss

```diff
@@ -27,7 +12,7 @@ $toggle-transition-animation: ease-in;
 
     .toggle-switch {
         display: inline-block;
-        height: $toggle-height;
+        height: $clr-toggle-height;
         margin-right: 1rem;
```

Comment:
Feels a bit weird that height has a variable assigned to it but margins and borders use values directly here. Would it make more sense to keep the variables in one file or just move the color variables in the variables file and use the size ones directly in the CSS? Otherwise we would have to jump around different files to figure out whats going on, no? Let me know what you think about this.